StardewArchipelago/Locations/CodeInjections/Modded/DeepWoodsModInjections.cs

```diff
@@ -82,6 +86,40 @@ public static class DeepWoodsModInjections
             }
         }*/
 
+        //public class DeepWoods
+        //public DeepWoods(DeepWoods parent, int level, EnterDirection enterDir)
+        public static void Constructor_WoodsDepthChecker_Postfix(GameLocation __instance, int level, Enum enterDir)
+        {
+            try
+            {
+                var deepWoodsStateType = AccessTools.TypeByName("DeepWoodsMod.DeepWoodsStateData");
```

Comment:
This particular part needs to be fixed.  This is a static method to pull this type in order to call it, but does not properly obtain the requested field.  Issue is that this is actually relying on save data and the type of data is of this type, which is currently stumping me.